pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```diff
@@ -1,21 +1,36 @@
 { ffmpeg_6-full
-, nv-codec-headers-12
+, ffmpeg
+, pipewire
+, openal
 , chromaprint
 , fetchFromGitHub
 , lib
+, fetchgit
 }:
 
-(ffmpeg_6-full.override {
-  nv-codec-headers-11 = nv-codec-headers-12;
-}).overrideAttrs (old: rec {
+(ffmpeg_6-full
+.override { # As upstream will not update to 6.1, they will wait until ffmpeg 7 come out
+  openal = openal.override {
+    pipewire = pipewire.override {
+      ffmpeg = ffmpeg.overrideAttrs {
+        src = fetchgit {
+          url = "https://git.ffmpeg.org/ffmpeg.git";
+          rev = "n6.0.1";
+          hash = "sha256-CalWjjjJpXs0qGlF4zqJps1k8OhCFkwjq8pHnjt6k30=";
+        };
+      };
+    };
+  };
```

Comment:
Why is this necessary?

Reply:
And does that cause any issues?

Reply:
Wait you mean the override or the new code? I already undid this, see @justinas comment.

Reply:
> And does that cause any issues?

I was able to build on x86_64-linux, and ofborg built on the other architectures. It failed on x86_64-linux saying that I hadn't specified the hash, but this is probably a bug.

Reply:
No, I mean the section I selected; overriding openal. This should not be necessary. Openal will use ffmpeg-headless (6.1) but I don't see why that should cause any issues.

Reply:
You're right, sorry!